pkg/chain/tbtc.go

```diff
@@ -95,12 +95,31 @@ type TBTCSystem interface {
 
 	// PastDepositRedemptionRequestedEvents returns all redemption requested
 	// events for the given deposit which occurred after the provided start block.
-	// All implementations should returns those events sorted by the
+	// All implementations should return those events sorted by the
 	// block number in the ascending order.
 	PastDepositRedemptionRequestedEvents(
 		startBlock uint64,
 		depositAddress string,
 	) ([]*DepositRedemptionRequestedEvent, error)
+
+	// FundingInfo retrieves the funding info for a particular deposit address
+	FundingInfo(
+		depositAddress string,
+	) (*FundingInfo, error)
+}
+
+// FundingInfo represents the funding information for a tbtc deposit
+type FundingInfo struct {
+	UtxoValueBytes [8]uint8
+	FundedAt       *big.Int
+	UtxoOutpoint   []uint8
+}
+
+// CreatedEvent is an event emitted when a deposit has been created.
+type CreatedEvent struct {
+	DepositAddress string
+	KeepAddress    string
+	BlockNumber    uint64
 }
```

Comment:
Do we still need it? Are we using it anywhere else besides local chain implementation?